docs/developer-docs/daos/nns/neuron-management.md

```diff
@@ -0,0 +1,160 @@
+
+### Interacting with neurons
+All interactions with a neuron go through the following API on the NNS governance cansiter. This includes everything in a neuron's lifetime such as the neuron's creation, modifications and voting, and finally unstaking the ICP tokens again from the neuron.
+```
+type ManageNeuron = record {
+  id : opt NeuronId;
+  command : opt Command;
+  neuron_id_or_subaccount : opt NeuronIdOrSubaccount;
+};
+```
+We will next to through a few important commands and how they can be used.
+
+
+### Creating a neuron and topping it up
+Staking ICP utility tokens in a neuron involves two commands:
+1. Sending ICP utility tokens to the neuron's subaccount.
+2. Claiming the neuron on the NNS governance canister, which basically means telling the governance canister that the transfer in 1. happened upon which NNS governance will create a new neuron.
+
+TODO: ADD DETAILS HOW THE SUBACCOUNT MUST BE COMPUTED!
+
+For the ledger transfer, the relevant `candid` interface can be found [here](https://github.com/dfinity/ic/blob/master/rs/rosetta-api/icp_ledger/ledger.did#L25) and contains the following details for a transaction:
+```
+type Transaction = record {
+    memo : Memo;
+    icrc1_memo: opt blob;
+    operation : opt Operation;
+    created_at_time : TimeStamp;
+}; 
+```
+To then claim the neuron, you can use the following command:
+TODO: WHICH WAY IS RECOMMENDED? ADD ARGUMENT EXPLANATION
+```
+type ClaimOrRefresh = record { by : opt By };
+type By = variant {
+  NeuronIdOrSubaccount : record {};
+  MemoAndController : ClaimOrRefreshNeuronFromAccount;
+  Memo : nat64;
+};
+type ClaimOrRefreshNeuronFromAccount = record {
+  controller : opt principal;
+  memo : nat64;
+};
+```
+Topping up an exising neuron with more tokens is called _refreshing_ a neuron and works in the same way. In a first step, additional tokens are sent to the neuron's account. In the second step, the NNS governance cansiter is informed so that it can update the neuron's stake after confirming with the ledger that there are new tokens. Note that refreshing of a neuron will also update the neuron's age to account for the fact that the newly added tokens have no age.
+
+TODO: ADD ARGUMENT EXPLANATION AND WHAT TO DO DIFFERENTLY FOR THE REFRESHING
+
+### Modifying a neuron's state
+
+* **Start dissolving**: The dissolve delay is like a kitchen timer that can only be turned in one direction. It can be arbitrarily increased, but only reduced by turning on dissolve mode and counting down. The neuron can be instructed to start “dissolving.” When the neuron is dissolving, its dissolve delay falls over time until it is either stopped or reaches zero. A neuron cannot vote (or earn rewards for voting) when its dissolve delay falls below six months. Once the dissolve delay reaches zero, it stops falling and the controlling principal can instruct the neuron to disburse.
```

Comment:
```suggestion
* **Start dissolving**: The neuron dissolve delay is like a kitchen timer that can only be turned in one direction. It can be arbitrarily increased, but only reduced by turning on dissolve mode and counting down. When the neuron is dissolving, its dissolve delay falls over time until it is either stopped or reaches zero. A neuron cannot vote (or earn rewards for voting) when its dissolve delay falls below six months. Once the dissolve delay reaches zero, it stops falling and the controlling principal can instruct the neuron to disburse.
```